### PR TITLE
[sailfish-components-webview] Fix for input prompts. Fixes JB#54845 OMP#JOLLA-231

### PR DESCRIPTION
**Pull request by mkenttala**
The binging doesn't work for some reason

### CODE REVIEW

File: import/popups/PromptDialog.qml [hunk 1]
```diff
@@ -18,7 +18,7 @@ Dialog {
     canAccept: input.text.length > 0
 
     property alias text: prompt.text
-    property alias value: prompt.value
+    property alias value: input.text
```
Comment (rainemak) [inline import/popups/PromptDialog.qml:21]: If "value: input.text" is not updated, then that should imply that prompt value that is send back to the engine [1] is not right either.

[1]
https://github.com/sailfishos/sailfish-components-webview/blob/master/import/popups/PopupOpener.qml#L189

Comment (mkenttala) [reply import/popups/PromptDialog.qml:21]: That PromptDialog.value property is actually what's sent to the server, without this server always receives nothing from javascript prompt method.

Comment (rainemak) [reply import/popups/PromptDialog.qml:21]: Yeap... and `CustomPromptPopup.qml` would need something like:
```
onAccepted: value = input.text
```
and removing that input.text binding.

Comment (rainemak) [reply import/popups/PromptDialog.qml:21]: Maybe even:
```
Binding { target: popup; property: "value"; value: input.text }
```

Tried to rename "value" property to promptValue => works as well.

**Review by rainemak — APPROVED** [review]
Note nicest fixes but these will indeed fix this. Hence, approving.

**Review by atatarov — APPROVED** [review]

**Pull request merged**